### ISSUE TITLE
setting plugins

### ISSUE THREAD

**Issue by mkhstar** [JiHong88/SunEditor#162]
Apart from loading all plugins, Setting plugin is not working it gives this error!
This is my code
```javascript
const suneditor = require('suneditor')
const align = require('suneditor/src/plugins/submenu/align')

suneditor.create('editor', {
  plugins: [align],
  buttonList: [['align']]
})


```
Uncaught Error: [SUNEDITOR.core.callPlugin.fail] The called plugin does not exist or is in an invalid format. (pluginName:"align")
    at Object.callPlugin (core.js:223)
    at HTMLDivElement.onClick_toolbar

**Comment by JiHong88** [comment]
Hi, @mkhstar 

The bug has been fixed.
I'll apply it this week's version update.
Thank you.

**Comment by JiHong88** [comment]
Is this the same issue? [#150](https://github.com/JiHong88/SunEditor/issues/150#issuecomment-562952360)

**Comment by JiHong88** [comment]
Hi,
Fixed in [v2.23.0](https://github.com/JiHong88/SunEditor/releases/tag/2.23.0)
Thank you.

**Comment by mkhstar** [comment]
Thanks
I applied it to suneditor-react
